.changeset/shy-turtles-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/kit": patch
+---
+
+fix: made `redirect` accept URL objects
```

Comment:
```suggestion
feat: accept `URL` in `redirect`
```

Reply:
Should I update `patch` to `minor` too?

Reply:
Yes :pray:

Reply:
following https://semver.org/ this is a minor, accepting URL is new functionality, or did we claim it supported it in docs?

Reply:
no, I don't believe we claimed to support it in the docs. it is a feature rather than a bug fix